src/beastie/commands/awesomeness/awesomeness.ts

```diff
@@ -1,8 +1,8 @@
-import { getTwitchId } from "../../../utils";
 import { BeastieLogger } from "../../../utils/Logging";
 import { CommandModule } from "../index";
 import CommandContext from "../utils/commandContext";
 import { getAwesomeness } from "../../../services/db";
+import { broadcaster } from "../../../services/twitch/TwitchAPI";
```

Comment:
*Associated with* ➡️ https://github.com/teamTALIMA/BeastieBot-V3/pull/13/files#r429017224
```suggestion
import { getTwitchProfile } from "../../../services/twitch/TwitchAPI";
```